docs/technical documentation/nanomi.md

```diff
@@ -0,0 +1,32 @@
+# NanoMi Optics Technical Documentation
+
+## System Architecture
+
+This software is contained in one large package with two other software: NanoMi Optics and Tomography Alignment Software. For each software, there is a GUI and an engine package. The engine package contains all of the core logic, processing, and math behind the software. The engine must have zero dependencies on the GUI packages and GUI libraries. The GUI package focuses on handling user interaction and displays. There are also two other packages: testing and common. The testing package holds all of the tests that are run on the software. The common package holds shared common functions that are reused across the software.
```

Comment:
this is outdated now since we have refactored, there is no a common package. And the tests aren't really in a package so to speak anymore.